lib/webvalve/monkey_patches.rb

```diff
@@ -0,0 +1,23 @@
+# if the webdrivers gem is included, patch it so that it can make HTTP requests to download
+# drivers when necessary.
+#
+# based on the v3.9.4 release of webdrivers gem
+# https://github.com/titusfortner/webdrivers/blob/v3.9.4/lib/webdrivers/network.rb#L47-L54
+#
+# and the v3.5.1 release of webmock gem
+# https://github.com/bblimke/webmock/blob/v3.5.1/lib/webmock/http_lib_adapters/net_http.rb#L12
+if defined?(Webdrivers)
```

Comment:
this has an implicit dependency on the webdrivers gem being loaded _before_ webvalve in your Gemfile. does anyone know of a trick to make that not matter?

Reply:
Can we require it in a non-raisey way?

Reply:
You could `include` or `prepend` this patch in `WebValve::Railtie`'s initialization block.

Reply:
> Or just require rescue LoadError, which seems short and to the point?

I'm thinking we can make it work if they are loaded in the other order too. But if we're not worried about that, then yeah, we can just catch the exception.

Reply:
> Or just require rescue LoadError, which seems short and to the point?

Oh, right, yeah that could work.